Makefile

```diff
@@ -17,7 +17,7 @@ CONCORD_BFT_CMAKE_FLAGS:= \
 			-DCMAKE_CXX_FLAGS_RELEASE=-O3 -g \
 			-DUSE_LOG4CPP=TRUE \
 			-DBUILD_ROCKSDB_STORAGE=TRUE \
-			-DUSE_S3_OBJECT_STORE=FALSE \
+			-DUSE_S3_OBJECT_STORE=TRUE \
```

Comment:
Is `TRUE` the same as `ON` ? Let's unify this everywhere.

Reply:
It's the same. What is the preferred naming?
1, On, Yes, True or 0, Off, No, False.

I use ON/OFF in cmake related toggles, but I'm not sure it's an industry standard.

Reply:
Thanks for the clarification! I don't have a strong opinion about this, as long as the values are interpreted correctly.